package.json

```diff
@@ -62,6 +59,7 @@
     "eslint-plugin-import": "^2.18.2",
     "file-loader": "^4.1.0",
     "if-env": "^1.0.0",
+    "mini-css-extract-plugin": "^0.8.0",
```

Comment:
I think `mini-css-extract-plugin` has to be added dependencies.

Reply:
It is required only at the building stage and not required for running the application